sormas-backend/src/test/java/de/symeda/sormas/backend/importexport/DatabaseExportServiceTest.java

```diff
@@ -62,6 +64,10 @@ public void testGetConfigFullyDefined() {
 	@Test
 	public void test_all_entities_have_export_configuration() {
 		Collection<String> exportableTables = DatabaseExportService.EXPORT_CONFIGS.values();
+		Set<DatabaseTable> joinTables = Arrays.stream(DatabaseTable.values()).filter(t -> t.getParentTable() != null).collect(Collectors.toSet());
+		Set<DatabaseTable> joinTablesFromExportConfigs =
+			DatabaseExportService.EXPORT_CONFIGS.keySet().stream().filter(t -> t.getParentTable() != null).collect(Collectors.toSet());
+
```

Comment:
This just tests that all items from DatabaseTable has an export config.

this tes checks the table names of entity classes, you should do similarly for join tables but the table names you get from  @JoinTable annotations of the getter methods

Reply:
Ok, sure. I've understood something else last time ( what I've implemented above)